src/lambdas/sayHello.js

```diff
@@ -0,0 +1,31 @@
+const AWS = require('aws-sdk');
+const {isOffline} = require('../helpers/env');
+const {post} = require('../helpers/http');
+
+exports.sayHello = (event, context, callback) => {
+    const { name } = JSON.parse(event.body);
+    if (!name) {
+        callback('No name received');
+    }
+
+    if (isOffline) {
```

Comment:
We can create a service called 'remoteLambda' with a function `invoke` and make this verification there.

Reply:
I'm gonna cover this part in another story